bootloaders/riotboot_dfu/doc.txt

```diff
@@ -55,8 +55,10 @@ $ FEATURES_REQUIRED+=riotboot USEMODULE+=usbus_dfu make -C examples/saul BOARD=p
 ```
 
 Note that when building and flashing a different slot (eg. `flash-slot1`),
-the DFU utility has to be explicitly prompted to upload it the other slot
-by adding a `DFU_ALT=1` argument.
+the DFU utility need to be explicitly prompted to upload it the other slot
```

Comment:
Now that this is automated, is there much point in still having this section? Then again, we could emphasize what changes:

```suggestion
not only is the image built for that slot, but also dfu-util gets passed `--alt 1` (via the `DFU_ALT` build variable) to store it in the right place.
```

(Is there any trick to make multi-line suggestions? I'm afrait this can't just be click-accepted.)